src/App.tsx

```diff
@@ -1,26 +1,234 @@
-/* eslint-disable max-len */
+/* eslint-disable jsx-a11y/label-has-associated-control */
 /* eslint-disable jsx-a11y/control-has-associated-label */
 import React from 'react';
-import { UserWarning } from './UserWarning';
-
-const USER_ID = 0;
+import { Todo } from './types/Todo';
+import { createTodo, deleteTodo, getTodos, updateTodo } from './api/todos';
+import { useEffect } from 'react';
+import classNames from 'classnames';
+import { TodoItem } from './components/TodoItem';
+import { ErrorNotification } from './components/ErrorNotification';
+import { Footer } from './components/Footer';
+import { FilterStatus } from './types/FilterStatus';
 
 export const App: React.FC = () => {
-  if (!USER_ID) {
-    return <UserWarning />;
+  const [todos, setTodos] = React.useState<Todo[]>([]);
+  const [errorMessage, setErrorMessage] = React.useState('');
+  const [status, setStatus] = React.useState(FilterStatus.all);
+  const [temptTodo, setTemptTodo] = React.useState<Todo | null>(null);
+  const [processingIds, setProcessingIds] = React.useState<number[]>([]);
+
+  const inputRef = React.useRef<HTMLInputElement>(null);
+
+  const [title, setTitle] = React.useState('');
+
+  let visibleTodos = todos;
+  const activeTodos = todos.filter(todo => !todo.completed);
+  const completedTodos = todos.filter(todo => todo.completed);
+
+  if (status === FilterStatus.active) {
+    visibleTodos = activeTodos;
+  } else if (status === FilterStatus.completed) {
+    visibleTodos = completedTodos;
   }
```

Comment:
- Create a function that filters todos and move it to utils/helpers folder/file
- Use switch instead